<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentManager</class>
 <widget class="QWidget" name="SegmentManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SegmentManager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3" rowspan="2">
    <widget class="ImageLabel" name="imageLabel2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="ImageLabel" name="imageLabel3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="ImageLabel" name="imageLabel0">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="ImageLabel" name="imageLabel1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QSvgWidget" name="segmentImage" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>212</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>../ImageLabel/ImageLabel.h</header>
  </customwidget>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header location="global">qsvgwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
